### PR TITLE
Documenation edits from recent onboarding

### PR DESCRIPTION
**Pull request by workingBen**
This includes documentation edits for:
* fixing file locations when `src` was omitted in the path
* fixing file locations for a hidden file and a jsx extension
* updating copy for a few sections
* update the `friends` server plugin use `response.data` (it was broken before this)
* config the server plugin to the correct location
* replace empty div with self closing div, so eslint passes

### CODE REVIEW

**Comment by DavidOrchard** [comment]
some of the edits may have been covered in https://github.com/electrode-io/electrode-io.github.io/pull/383

**Comment by workingBen** [comment]
@DavidOrchard Yeah, I think we both fixed a couple `src` paths. I'm fine with yours going in first and rebasing this PR, or just closing your PR. I saw you also added Notifications in your PR, but there is a comment about that being potentially extraneous. Let me know how you'd like to handle this duplication.

**Comment by DavidOrchard** [comment]
How about you clear out your edits that duplicate mine.

File: _docs/03.2-build_server_plugin.markdown [hunk 1]
```diff
@@ -99,7 +99,7 @@ exports.register = (server, options, next) => {
     };
     return githubGetContributors(githubInfo)
       .then((response) => {
-        friend.friends = response.map(({ login, avatar_url, html_url }) => (
+        friend.friends = response.data.map(({ login, avatar_url, html_url }) => (
```
Comment (donsalari) [inline _docs/03.2-build_server_plugin.markdown]: 👍

**Comment by donsalari** [comment]
LGTM

**Pull request merged**